src/darshana/UnionofArrays.java

```diff
@@ -0,0 +1,37 @@
+package darshana;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+ 
+public class UnionofArrays {
```

Comment:
No Darshna,
you need to complete this assignment without using collection framework.